README.md

```diff
@@ -50,80 +50,108 @@ The DeepSparse Engine is a CPU runtime that delivers unprecedented performance b
 
 This repository includes package APIs along with examples to quickly get started learning about and actually running sparse models.
 
-### Related Products
+## Sparsification
 
-- [SparseZoo](https://github.com/neuralmagic/sparsezoo):
-  Neural network model repository for highly sparse models and optimization recipes
-- [SparseML](https://github.com/neuralmagic/sparseml):
-  Libraries for state-of-the-art deep neural network optimization algorithms,
-  enabling simple pipelines integration with a few lines of code
-- [Sparsify](https://github.com/neuralmagic/sparsify):
-  Easy-to-use autoML interface to optimize deep neural networks for
-  better inference performance and a smaller footprint
+Sparsification is the process of taking a trained deep learning model and removing redundant information from the over precise and over parameterized network resulting in a faster and smaller model.
+Techniques for sparsification are all encompassing including everything from inducing sparsity using [pruning](https://neuralmagic.com/blog/pruning-overview/) and [quantization](https://arxiv.org/abs/1609.07061) to enabling naturally occurring sparsity using [activation sparsity](http://proceedings.mlr.press/v119/kurtz20a.html) or [winograd/FFT](https://arxiv.org/abs/1509.09308). 
+When implemented correctly, these techniques result in significantly more performant and smaller models with limited to no effect on the baseline metrics.
+For example, pruning plus quantization can give over [7x improvements in performance](resnet50link) while recovering to nearly the same baseline.
```

Comment:
nit: placeholder link [7x improvements in performance](resnet50link)

Reply:
waiting on the link to be provided to include here